### PR TITLE
create nx_version.txt #2, nx_dev_test_output.txt #3, Run timing scripts from nx-parallel #191 and creating a python script nx_tutorial_script.py #5

### PR DESCRIPTION
**Pull request by tiwavaldese**
created nx_version.txt #2 , nx_dev_test_output.txt #3 , Run timing scripts from nx-parallel #191 and creating a python script nx_tutorial_script.py #5

### CODE REVIEW

**Comment by tiwavaldese** [comment]
create nx_version.txt #2 , nx_dev_test_output.txt #3, Run timing scripts from nx-parallel #191 and creating a python script nx_tutorial_script.py #5

File: 2023-round-2/tiwavaldese/nx_version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+print(nx.__version__)
```
Comment (MridulS) [inline 2023-round-2/tiwavaldese/nx_version.txt:1]: Could you put the output of this command? Thanks!

Comment (tiwavaldese) [reply 2023-round-2/tiwavaldese/nx_version.txt]: ok i will

**Comment by tiwavaldese** [comment]
please help review my work **Setup local development environment for NetworkX #3**

File: 2023-round-2/tiwavaldese/nx_dev_test_output.txt [hunk 2]
```diff
@@ -0,0 +1,1083 @@
...
+  C:\Users\00\networkx\networkx\linalg\algebraicconnectivity.py:302: UserWarning: Exited postprocessing with accuracies
+  [0.00056623]
+  not reaching the requested tolerance 1e-08.
+    sigma, X = sp.sparse.linalg.lobpcg(
+
+-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
+=============================================== short test summary info ===============================================
+FAILED networkx/networkx/drawing/tests/test_pydot.py::test_pydot_issue_258 - FileNotFoundError: [WinError 2] "neato" not found in path.
+FAILED networkx/networkx/drawing/tests/test_pydot.py::test_pydot_numerical_name - FileNotFoundError: [WinError 2] "dot" not found in path.
+==================== 2 failed, 5092 passed, 56 skipped, 3 xfailed, 7 warnings in 282.04s (0:04:42) ====================
```
Comment (MridulS) [inline 2023-round-2/tiwavaldese/nx_dev_test_output.txt:1083]: Thanks for this! The tests are failing. Could you look into this? You can also try removing pydot so pytest skips the tests.

Comment (tiwavaldese) [reply 2023-round-2/tiwavaldese/nx_dev_test_output.txt]: Thank you @MridulS it pass now without any fail.

please does it means i have done my first contribution already?

Comment (MridulS) [reply 2023-round-2/tiwavaldese/nx_dev_test_output.txt]: Can you please update the test output file with the new output then? Thanks!

Comment (tiwavaldese) [reply 2023-round-2/tiwavaldese/nx_dev_test_output.txt]: yeah i did already and push

**Comment by MridulS** [comment]
Thanks!

**Comment by tiwavaldese** [comment]
thanks too

**Pull request merged**